app/models/reports/result.rb

```diff
@@ -93,10 +98,12 @@ def last_key(key)
     end
 
     [:period_info, :earliest_registration_period,
-      :registrations, :registrations_with_exclusions,
-      :adjusted_registrations, :cumulative_registrations,
+      :registrations, :denominator_registrations,
+      :ltfu_patients, :cumulative_registrations,
+      :adjusted_registrations, :adjusted_registrations_with_ltfu,
       :missed_visits, :missed_visits_rate,
-      :controlled_patients, :controlled_patients_rate,
+      :controlled_patients, :controlled_patients_with_ltfu,
+      :controlled_patients_rate, :controlled_patients_with_ltfu_rate,
```

Comment:
I'm sensing some simplification in this area.
* Aren't controlled patients and controlled patients with LTFU the same? If you're controlled, you've had a BP in the last 3 months, you cannot be LTFU.
* The same would apply for uncontrolled patients
* The total patients (denominator) is what would switch between with or without LTFU

Hmm, here's the amorphous idea in my head. We store the following datapoints in the result. Each point would be an array of 24 numbers for the last 24 months.
1. Dead patients
2. LTFU patients
3. Not LTFU patients
4. Controlled patients
5. Uncontrolled patients

1 + 2 + 3 should be the total registered, right?

4 / (1 + 2 + 3) is our current control rate

4 / 3 is the new exclusioned control rate

5 / (1 + 2 + 3) is our current control rate

5 / 3 is the new exclusioned control rate

(1 + 2 +3) - (4 + 5) is current missed visit count, and that / (1 + 2 + 3) is the current missed visit rate

(3) - (4 + 5) is the new missed visit count, and that / (3) is the new missed visit rate.

Are these ideas already implemented? Do I have incorrect assumptions? I'd love to have you walk me through this 😅